clang/test/dpct/module_wrapper_gen.cu

```diff
@@ -0,0 +1,26 @@
+// RUN: dpct --extra-arg="--ptx" \
+// RUN:      --out-root=%T/module_wrapper_gen \
+// RUN:      --cuda-include-path="%cuda-path/include" %s
+// RUN: FileCheck %s --input-file=%T/module_wrapper_gen/module_wrapper_gen.dp.cpp
+
+// START
+
```

Comment:
Can we add a functions:

__device__
void int_test() {  // check non-extern-C device non-operator is not wrapped
}

__global__
void int_wow() { // check non-extern-C global is not wrapped
}

to ensure that these are not wrapped.

Reply:
I've added these cases in the test